templates/experiments/experiment_form.html

```diff
@@ -81,6 +81,18 @@ <h3 class="text-md font-bold">{% translate "Please note" %} <i class="fas fa-exc
         {% render_form_fields form "tools" %}
       </div>
 
+      <input type="radio" name="main_tabs" role="tab" class="tab" aria-label="Allowlist">
+      <div role="tabpanel" class="tab-content">
+        <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
+        <p class="py-4 text-sm">
+          {% blocktranslate %}
+            To limit the set of particiapants that can chat with this bot, simply add their identifiers in this list.
+            When the list is empty, the bot will be public.
+          {% endblocktranslate %}
+          {% render_form_fields form "participant_allowlist" %}
```

Comment:
Would it make sense to use the TomSelect widget here?

Reply:
I was also thinking about that but I'd prefer a table of sorts to that list. But this list would be better than the current input element. It's not a big lift, so I can add this quickly and maybe down the line add better UI + support for CSV uploads

Reply:
I'm torn between two: 1. do something quick and easy and 2. do it properly from the get-go

Reply:
ah good point. Didn't think of that. I'll look into that as well

Reply:
https://github.com/dimagi/open-chat-studio/pull/625/commits/b52e2e05d8e21b93587bc5f6176a9cad0d3db71d

Tom select supports pasting, but not copying, so I added a copy button.

![image](https://github.com/user-attachments/assets/bac69525-9e54-403e-b037-8c5d25fb0ff8)